src/Session.hs

```diff
@@ -92,10 +92,10 @@ isSuccess :: Maybe Summary -> Bool
 isSuccess = not . isFailure
 
 parseSummary :: String -> Maybe Summary
-parseSummary r = case reverse $ lines r of
-  x : _ | Just summary <- readMaybe (dropAnsiEscapeSequences x) -> Just summary
-  _ -> Nothing
+parseSummary = findJust . (map $ readMaybe . dropAnsiEscapeSequences) . take 3 . reverse . lines
```

Comment:
If any of the lines can be parsed as a summary even though they are not, the test result will be wrong.  In order to minimize that from happening, I limited the window that is examined.

I admit that this is an esoteric problem, so if you poke me once more I'll drop the `take 3` without further discussion.